content/general/firehose/sqs-rule.textile

```diff
@@ -0,0 +1,73 @@
+---
+title: Amazon SQS rule
+meta_description: "Ably allows you to send data to SQS based on message, channel lifecycle, channel occupancy, and presence events."
+meta_keywords: "SQS, integrations, events, serverless"
+section: general
+index: 10
+languages:
+  - none
+jump_to:
+  Help with:
+    - Development status#development-status
+    - Creating a SQS rule#creating-sqs-rule
+    - Creating a SQS rule in the dashboard#creating-rule-dashboard
+    - Creating a SQS rule using Control API#creating-rule-control-api
+    - See also#see-also
+---
+
+You can use a SQS rule to send "data":/general/firehose#data-sources such as messages, occupancy, lifecycle and presence events from Ably to SQS.
+
+h2(#development-status). Development status
+
+This feature is "Enterprise only":https://ably.com/pricing, and currently in Preview.
+
+h2(#creating-sqs-rule). Creating a SQS rule
+
+There are two ways to create a SQS rule:
+
+1. Using the "Ably Dashboard":https://ably.com/dashboard.
+2. Using the "Control API":/control-api.
+
+h3(#creating-rule-dashboard). Creating a SQS rule in the dashboard
+
+To create a rule in your "dashboard":https://ably.com/dashboard:
+
+1. Login and select the application you wish to integrate with SQS.
+2. Click the *Integrations* tab.
+
+<a href="/images/screenshots/sqs-rule/integrations-tab.png" target="_blank">
+  <img src="/images/screenshots/sqs-rule/integrations-tab.png" style="width: 100%" alt="Integrations tab">
+</a>
+
+3. Click the *+ New Integration Rule* button.
+4. Choose Firehose.
+5. Choose AWS SQS.
+
+<a href="/images/screenshots/sqs-rule/choose-sqs.png" target="_blank">
+  <img src="/images/screenshots/sqs-rule/choose-sqs.png" style="width: 100%" alt="Choose sqs">
+</a>
+
+6. Configure the settings applicable to your use case and your SQS installation.
+
+The SQS-specific settings:
+
+<a href="/images/screenshots/sqs-rule/sqs-rule-settings-sqs.png" target="_blank">
+  <img src="/images/screenshots/sqs-rule/sqs-rule-settings-sqs.png" style="width: 100%" alt="sqs rule settings - sqs-specific">
+</a>
+
+The general settings:
+
+<a href="/images/screenshots/sqs-rule/sqs-rule-settings-general.png" target="_blank">
+  <img src="/images/screenshots/sqs-rule/sqs-rule-settings-general.png" style="width: 100%" alt="sqs rule settings - general">
+</a>
+
+7. Click *Create* to create the rule.
+
+h3(#creating-rule-control-api). Creating a SQS rule using Control API
+
+See some examples of creating rules in the "rules section":/control-api/curl-examples#examples-rules of the Control API documentation. You can also refer to the "rule section":/control-api/api-reference#tag/rules/paths/~1apps~1{app_id}~1rules/post of the Control API Reference for information on creating a SQS rule. You will need to select a @ruleType@ of @aws/SQS@.
```

Comment:
> You will need to select a @ruleType@ of @aws/SQS@.

I think the rule type can just be `aws/sqs` - where SQS is actually lower case, as that's how it appears in the dropdown in the reference docs.